CONTRIBUTING.md

```diff
@@ -89,6 +89,28 @@ To install dependencies once you have Node and yarn installed, run this in the p
 $ yarn
 ```
 
+## Adding a Plugin
+
+To create a new plugin for a third-party package, follow these steps:
+
+1. ``mkdir -p packages/datadog-plugin-<pluginname>/src``
+2. ``cp packages/datadog-plugin-kafkajs/src/index.js packages/datadog-plugin-<pluginname>/src``
```

Comment:
I would probably add a small note about why `kafkajs`, eg:

```suggestion
2. Copy an `index.js` file from another plugin to use as a starting point: `cp packages/datadog-plugin-kafkajs/src/index.js packages/datadog-plugin-<pluginname>/src` 
```

Reply:
But in reality, I don't think copying is such a good idea. It would be better to have a scaffold plugin or something like that.